life_expectancy/main.py

```diff
@@ -0,0 +1,22 @@
+"""
+Main script for life_expectancy module
+"""
+
+from pathlib import Path
+from life_expectancy.data_loading import load_data, save_data
+from life_expectancy.data_cleaning import clean_data
+
+PROJECT_DIR = Path(__file__).parent
+FILE_PATH = f"{PROJECT_DIR}/data/eu_life_expectancy_raw.tsv"
+SAVE_PATH = "life_expectancy/data/pt_life_expectancy.csv"
```

Comment:
this variable can't also use the PROJECT_DIR variable?
Like: `SAVE_PATH = PROJECT_DIR / data / pt_life_expectancy.csv`